hive-metastore/src/main/java/org/apache/iceberg/hive/HiveViewOperations.java

```diff
@@ -299,7 +299,10 @@ private void setHmsTableParameters(
           BaseMetastoreTableOperations.PREVIOUS_METADATA_LOCATION_PROP, currentMetadataLocation());
     }
 
-    setSchema(metadata.schema(), parameters);
+    HMSTablePropertyHelper.setSchema(
```

Comment:
I know the intention of this PR, but for me the refactor seems half done in case we didn't move the `HiveViewOperations.setHmsTableParameters()` also to the util class. With the current state we have a util class to give us the params required for HMS but it only does it for tables, and partially for views. I'd find it beneficial to include views too.

Reply:
I would suggest to keep the View and the Table code as similar as possible.
Maybe Hive doesn't use Iceberg views ATM, but I prefer to keep the 2 codepath consistent

Reply:
Moved HiveViewOperations.setHmsTableParameters() to HMSTablePropertyHelper.